bundles/org.openhab.binding.ojelectronics/src/main/resources/OH-INF/i18n/ojelectronics.properties

```diff
@@ -53,3 +51,8 @@ channel-type.ojelectronics.thermostatName.label = Thermostat Name
 channel-type.ojelectronics.vacationBeginDay.label = Vacation Begin Day
 channel-type.ojelectronics.vacationEnabled.label = Vacation Mode Enabled
 channel-type.ojelectronics.vacationEndDay.label = Vacation End Day
+
+# thing types config
+
+thing-type.config.ojelectronics.ojcloud.refreshDelayInSeconds.label = Refresh Delay
+thing-type.config.ojelectronics.ojcloud.refreshDelayInSeconds.description = Refresh delay in seconds.
```

Comment:
This can now be removed, it should not reappear when regenerating the file using the tool.